code/evaluation/score_models.r

```diff
@@ -63,15 +63,20 @@ report_dates <- seq(start_date, latest_date, by = "week")
 
 for (chr_report_date in as.character(report_dates)) {
   report_date <- as.Date(chr_report_date)
-  filename <-
+  eval_filename <-
     here::here("evaluation", paste0("evaluation-", report_date, ".csv"))
 
-  table <- score_models(
-    data,
-    report_date,
-    restrict_weeks = 4,
+  scores <- score_forecasts(
```

Comment:
It doesn't seem super efficient to include this in the loop. In the case where we have to re-run all the evaluations (which is precisely what we will have to do once this is merged), we are going to repeat the same operations many times since scores(t+1) are just a superset of scores(t).

Reply:
Yes, good point - in fact I already did this in [this experimental branch](https://github.com/epiforecasts/covid19-forecast-hub-europe/blob/a2c232f796a7b7baff21872c9f9d89553b090e1f/code/evaluation/score_models.r#L60). I'll copy over the change from there.